keras/backend/mxnet_backend.py

```diff
@@ -375,22 +393,28 @@ def placeholder(shape=None, ndim=None, dtype=None, sparse=False, name=None):
         placeholder1:[tensor=False dtype=float32]
     ```
     """
-    if sparse:
-        raise NotImplementedError('MXNet backend do not yet support sparse tensor operations.')
-
     if dtype is None:
         dtype = floatx()
 
     dtype = _convert_string_dtype(dtype)
     if shape is None and ndim is None:
         raise ValueError('MXNet Backend: Specify either a shape or ndim value.')
     name = _prepare_name(name, 'placeholder')
+
     if shape:
         shape = tuple([0 if dim is None else dim for dim in shape])
     else:
         shape = tuple([0 for _ in range(ndim)])
 
-    sym = _keras_variable(name, shape=shape, dtype=dtype)
+    if sparse:
+        sym = _keras_variable(name, shape=shape, dtype=dtype, stype='csr')
+        sym._keras_shape = tuple([d if d != 0 else None for d in shape])
+        sym._mxnet_placeholder = True
+        sym._uses_learning_phase = False
+        print(sym)
```

Comment:
I think you missed to remove this print.

Reply:
removed